docs/installation/installation-using-docker-macos.md

```diff
@@ -27,42 +27,56 @@ There are two domains each for different language in default installation. First
 sudo ifconfig lo0 alias 127.0.0.2 up
 ```
 
-### 2. Create docker-compose.yml file
+### 2. Create docker-compose.yml and docker-sync.yml
 Create `docker-compose.yml` from template [`docker-compose-mac.yml.dist`](../../project-base/docker/conf/docker-compose-mac.yml.dist).
 ```
 cp docker/conf/docker-compose-mac.yml.dist docker-compose.yml
 ```
 
-### 3. Compose Docker container
-On MacOS you need to synchronize folders using docker-sync.
-Before starting synchronization you need to create a directory for persisting Postgres data so you won't lose it when the container is shut down.
-```
-mkdir -p var/postgres-data
-docker-sync start
-```
+Create `docker-sync.yml` from template [`docker-sync.yml.dist`](../../project-base/docker/conf/docker-sync.yml.dist).
+
+### 2.1 Set the UID and GID to allow file access in mounted volumes
+Because we want both the user in host machine (you) and the user running php-fpm in the container to access shared files, we need to make sure that they both have the same UID and GID.
+This can be achieved by build arguments `www_data_uid` and `www_data_gid` that should be set to the UID and GID as your own user in your `docker-compose.yml`.
+Also, you need to change `sync_userid` in `docker-sync.yml` file.
+
+You can find out your UID by running `id -u` and your GID by running `id -g`.
+
+#### 2.1 Fill your credentials into files
```

Comment:
This 2.1 section is basically a duplicate of the 2.1 section above. I would remove it and keep the upper version as it feels more natural to me and it's shorter.

See https://github.com/shopsys/shopsys/blob/82823de1d3e518011f68110f04834e56f3aba584/docs/installation/installation-using-docker-macos.md#21-set-the-uid-and-gid-to-allow-file-access-in-mounted-volumes